scripts/cleaning/delete-dotfiles.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 # Remove old deployment states
-rm -rf **/.serverless
+find . -name "*.serverless" -type d -exec rm -rf {} +
```

Comment:
why the change?

Reply:
For some unknown reason, it was not working for me in current os. (manjaro kde)
Previously it was, but yesterday it took me an hour to understand why env vars were not correct even after delete and re-deploy.

Reply:
that's weird :/